vector/src/main/res/layout/fragment_space_leave_advanced.xml

```diff
@@ -16,41 +16,107 @@
+        <com.google.android.material.appbar.CollapsingToolbarLayout
+            android:id="@+id/spaceExploreCollapsingToolbarLayout"
             android:layout_width="match_parent"
-            android:layout_height="?attr/actionBarSize"
-            android:minHeight="0dp"
-            app:title="@string/pick_tings_to_leave"
-            app:layout_scrollFlags="scroll|exitUntilCollapsed|snap|enterAlways"/>
+            android:layout_height="match_parent"
+            app:contentScrim="?android:colorBackground"
+            app:layout_scrollFlags="scroll|exitUntilCollapsed|enterAlways|snap"
+            app:scrimAnimationDuration="250"
+            app:scrimVisibleHeightTrigger="120dp"
+            app:titleEnabled="false"
+            app:toolbarId="@id/toolbar">
 
-        <androidx.appcompat.widget.SearchView
-            android:id="@+id/publicRoomsFilter"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="8dp"
-            android:layout_marginBottom="8dp"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/addRoomToSpaceToolbar"
-            app:queryHint="@string/search_hint_room_name" />
+            <androidx.appcompat.widget.LinearLayoutCompat
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="?attr/actionBarSize"
+                android:minHeight="0dp"
+                android:orientation="vertical"
+                android:paddingHorizontal="16dp">
+
+                <TextView
+                    android:id="@+id/spaceLeavePromptTitle"
+                    style="@style/TextAppearance.Vector.Body.Medium"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="24dp"
+                    android:text="@string/space_leave_prompt_msg_with_name" />
+
+                <TextView
+                    android:id="@+id/spaceLeavePromptDescription"
+                    style="@style/TextAppearance.Vector.Body"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/space_leave_prompt_msg_only_you"
+                    android:textColor="?vctr_content_secondary" />
+
+                <TextView
+                    android:id="@+id/spaceLeaveRadioButtonsTitle"
+                    style="@style/TextAppearance.Vector.Subtitle.Medium"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="16dp"
+                    android:text="@string/space_leave_radio_buttons_title"
+                    android:textAllCaps="true"
```

Comment:
Just a remark on the design, it's weird to have user content (the space name) capitalized. I would avoid that.

<img width="280" alt="image" src="https://user-images.githubusercontent.com/3940906/167004921-c0f0552e-dc47-4a7a-b9ce-2fddf169ec57.png">

Reply:
@niquewoodhouse could you confirm if we want it to be all caps?

Reply:
Thanks @bmarty for the comment, you're right.  @fedrunov could we amend the content to be `THINGS IN THIS SPACE`?  The caps styling comes from the treatment of subtitles throughout the app so I don't want to move away from it.  Thanks